.changeset/nervous-tigers-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wagmi/connectors": patch
```

Comment:
You also want to add `wagmi` and `@wagmi/core` to this list so they get the update downstream too!

Reply:
Done 👍